src/main/webapp/jsp/instructorFeedbacks.jsp

```diff
@@ -696,6 +696,7 @@
                 }
             %>
         </table>
+        <p align-center>Note: The table above doesn't contain sessions from archived courses. To view sessions from an archived course, unarchive the course first.</p>
```

Comment:
I'm not sure if there's an align-center html attribute. In your previous pull request, I mentioned to use the align-center *css class*.